Translate GDPR Cheat Sheet

xeluna opened the pull request:
#93, since I think no one is working on it.

Let me know if there's other changes to the metadata required. There currently isn't a field specifically for the translator so I didn't add my name.

Any suggestions on how to word the translations better are welcome.

xeluna commented on content/blog/gdpr-cheat-sheet.en.md:
```diff
@@ -0,0 +1,139 @@
+* Behaviour
+* Location or movements
+
+A classic example is the mobile phone or credit card credit check that you need to pass before getting a contract. Many companies will forward your data to credit agencies such as [Schufa]({{< ref "schufa" >}}), [Creditreform Boniversum]({{< ref "creditreform-boniversum" >}}) or [infoscore]({{< ref "infoscore" >}}). These agencies will then determine how likely you are to fulfil your contractual  ...
```
Do you want these examples replaced with some non-German examples?

baltpeter commented:
Thank you, will take a closer look when I have the time. As for your two questions:

* I don't think we need to differentiate between the author and translator. Just add your name (alias is fine, of course) and an optional link to `config/_default/params.toml` and then reference that in the `authors` key of the post. :)
* Please do replace German examples with better alternatives for an English post (i.e. UK-specific or international ones, I suppose), yes.

baltpeter commented on content/blog/gdpr-cheat-sheet.en.md:
```diff
@@ -0,0 +1,139 @@
+
+### Data Protection Officer
+
+A data protection officer's task is to ensure that a company (or another organisation, such as an authority) complies with data protection law. He is appointed by the company itself and must have a certain degree of independence in order to perform his duties.
```
The term `authority` may be a little misleading here. We usually use `public body` (although technically, that has a bit of a different meaning). `Government agency` also works fine.

xeluna replied:
I went to look at how it was phrased and [here](https://gdpr-info.eu/art-37-gdpr/), it uses `public authority`. I did more research and it seems that the GDPR itself does not specify what this entails, and it is up to member states to define it exactly.

For the UK, it's broader than government agencies only, and using that might narrow the definition too much. Would it be too technical to leave it as `public authority`?

baltpeter replied:
`public authority` is definitely fine, too. I just want to avoid confusion with the supervisory authorities (although, to be fair, they will have their own DPOs, too).

xeluna replied:
I decided to use `government agency` in the end, since, after all, it's meant to be one example. This would definitely make a clearer distinction from supervisory authorities.

baltpeter commented on content/blog/gdpr-cheat-sheet.en.md:
```diff
@@ -0,0 +1,139 @@
+
+The concept of personal data is the crux of the GDPR. Almost all of the regulations refer to it. As soon as information can be assigned to a specific person, it is considered personal data. A name is not necessarily required for the assignment. For example, an IP address, a user name, a cookie, or an identification number would suffice.
+
+The information that the IP address 31.41.59.26 called our [generator]({{< ref "generator" >}}) on 25 May 2018 would therefore be considered personal data. On the other hand, the information that on the same day in a bakery a third of the customers paid with cash is not.
```
Might just be me but I would associate calling with using a phone. :D How about `visited`?

xeluna replied:
`visited` would definitely work better, I was thinking too technically. :P

baltpeter commented on content/blog/gdpr-cheat-sheet.en.md:
```diff
@@ -0,0 +1,139 @@
+
+You will find a comprehensive overview and a detailed explanation on this subject in our [article on your GDPR rights]({{< ref "your-gdpr-rights" >}}).
+
+### Data Controller
```
Not the first instance where this is relevant but here's where I noticed it: In the German post, I sorted the heading alphabetically. We don't need to do that here, though. I'll leave the decision to you.

xeluna commented:
Here's a summary of the changes:
- Changed headings to sentence case
- Sorted terms alphabetically
- Replaced the credit agencies example with (UK's) Experian and Equifax
- Other suggestions from @baltpeter

baltpeter commented:
Awesome, let's get this merged.
